packages/react-native-editor/ios/Podfile

```diff
@@ -28,7 +28,7 @@ end
 
 target 'GutenbergDemo' do
   # Comment the next line if you don't want to use dynamic frameworks
-  use_frameworks!
+  use_frameworks! linkage: :static
```

Comment:
Without this it was failing, it replicates the same usage from the XCFramework see its [Podfile config](https://github.com/wordpress-mobile/gutenberg-mobile/blob/trunk/ios-xcframework/Podfile#L66).